### PR TITLE
testing/openjdk9: New aport

### PR DESCRIPTION
**Pull request by bratkartoffel**
This PR adds openjdk9 which is required to build openjdk10 (coming soon)
As the IcedTea project hasn't released any Java8+ buils so far this
package is built upon the oracle openjdk.

The "build.patch" is based loosely on the diff published at
https://builds.shipilev.net/patch-openjdk-portola-portola/
and have been backported to current JDK9.

This aport is only the first of a series I currently plan to contribute. I'd really like to have openjdk11 (LTS) in alpine, but sadly you should build JDK11 with JDK10, which needs JDK9 to build.

Output from java --version:
```
openjdk 9.0.4
OpenJDK Runtime Environment (build 9.0.4+12-alpine-r0)
OpenJDK 64-Bit Server VM (build 9.0.4+12-alpine-r0, mixed mode)
```

Architectures I successfully cross-compiled so far:
- [x] x86
- [x] x86_64
- [x] armv7
- [x] armhf
- [x] aarch64
- [x] ppc64le
- [x] s390x

I deployed the built packages on my server, you can directly download the apk-files when you add ```https://alpine.fraho.eu/testing``` to your `/etc/apk/repositories`. The corresponding public key can be found at `https://alpine.fraho.eu/build-5b76c816.rsa.pub`.

I've also created a docker container (only x86_64 so far) containing the new aport: https://hub.docker.com/r/bratkartoffel/openjdk/tags

### CODE REVIEW

File: community/openjdk9/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+		--disable-warnings-as-errors \
+		--disable-precompiled-headers \
+		--enable-dtrace=no \
+		--with-jvm-variants=server \
+		--with-debug-level=release \
+		--with-version-opt=alpine \
+		--with-version-pre= \
+		--with-version-minor=0 \
+		--with-version-security=0 \
+		--with-version-build=181
```
Comment (bratkartoffel) [inline community/openjdk9/APKBUILD]: we could extract that from $pkgver

**Comment by bratkartoffel** [comment]
I think that aport is ready to merge from my side.

**Comment by bratkartoffel** [comment]
s390x with qemu doesn't run:
```
/ # java -version
OpenJDK 64-Bit Server VM warning: Can't detect initial thread stack location - find_vma failed
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  Internal Error (vm_version_s390.hpp:367), pid=27133, tid=27137
#  guarantee((_features[0] & GnrlInstrExtFacilityMask) == GnrlInstrExtFacilityMask) failed: We no more support older than z10.
#
# JRE version:  (9.0+12) (build )
# Java VM: OpenJDK 64-Bit Server VM (9.0.4+12-alpine-r0, mixed mode, tiered, compressed oops, g1 gc, linux-s390x)
# Core dump will be written. Default location: core.27133 (may not exist)
#
# An error report file with more information is saved as:
# //hs_err_pid27133.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
#
Illegal instruction
```

**Comment by bratkartoffel** [comment]
JTReg tests don't look that bad:
```
==============================
Test summary
==============================
   TEST                                              TOTAL  PASS  FAIL ERROR
>> jtreg:jdk/test:tier1                               1645  1642     3     0 <<
>> jtreg:langtools/test:tier1                         3907  3900     7     0 <<
   jtreg:nashorn/test:tier1                              0     0     0     0
   jtreg:jaxp/test:tier1                                 0     0     0     0
==============================
TEST FAILURE
==============================
Test summary
==============================
   TEST                                              TOTAL  PASS  FAIL ERROR
>> jtreg:jdk/test:tier1                               1645  1642     3     0 <<
>> jtreg:langtools/test:tier1                         3907  3900     7     0 <<
   jtreg:nashorn/test:tier1                              0     0     0     0
   jtreg:jaxp/test:tier1                                 0     0     0     0
==============================
TEST FAILURE==============================
Test summary
==============================
   TEST                                              TOTAL  PASS  FAIL ERROR
>> jtreg:jdk/test:tier1                               1645  1642     3     0 <<
>> jtreg:langtools/test:tier1                         3907  3900     7     0 <<
   jtreg:nashorn/test:tier1                              0     0     0     0
   jtreg:jaxp/test:tier1                                 0     0     0     0
==============================
TEST FAILURE
```
I'll take a look at this

**Comment by bratkartoffel** [comment]
I'd really love to continue working on the JDK 10 and 11 aports, can you please merge this one?

**Comment by J0WI** [comment]
I've tested this successfully on `x86_64`.

**Comment by J0WI** [comment]
Btw., if you rebase you should get a build on Drone CI.

**Pull request closed without merging**